### PR TITLE
Create static.yml

### PR DESCRIPTION
**Pull request by ethan-w-foshee**
I think this workflow is set up right? Does it look right to you?

### CODE REVIEW

File: .github/workflows/static.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Simple workflow for deploying static content to GitHub Pages
+name: Deploy static content to Pages
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["dev"]
```
Comment (CyborgPotato) [inline .github/workflows/static.yml]: We'll want to do it on "dev" and "main"

File: .github/workflows/static.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+jobs:
+  # Single deploy job since we're just deploying
+  deploy:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```
Comment (CyborgPotato) [inline .github/workflows/static.yml]: We'll have two checkouts, one for main, one for dev like:

```yaml
- name: Checkout Main
  uses: actions/checkout@v4
- name: Checkout Dev
  uses: actions/checkout@v4
  ref: dev
  path: dev/
```

**Review by CyborgPotato — COMMENTED** [review]
We want it to deploy the up to date static website on changes ot main or dev

File: .github/workflows/static.yml (hunk 1 above)
Comment (ethan-w-foshee) [reply .github/workflows/static.yml]: Changes made?

File: .github/workflows/static.yml (hunk 2 above)
Comment (ethan-w-foshee) [reply .github/workflows/static.yml]: See above?

**Review by CyborgPotato — APPROVED** [review]
LGTM!

**Pull request merged**